packages/create/src/actions/project-name.ts

```diff
@@ -0,0 +1,44 @@
+import fs from 'node:fs';
+import { text, confirm, log, isCancel } from '@clack/prompts';
+import color from 'picocolors';
+import { cancelStep } from '../utils';
+import { Context } from '..';
+
+export const projectName = async (ctx: Context) => {
+  const projectName = await text({
+    message: "What's your project name? We'll create a directory for you.",
+    defaultValue: './new-keystatic-project',
+    placeholder: './new-keystatic-project',
+  });
+
+  if (isCancel(projectName)) cancelStep();
+
+  if (typeof projectName === 'string') {
+    ctx.projectName = projectName;
+
+    if (projectName === '.' || projectName === './') {
+      ctx.cwd = process.cwd();
+    } else {
+      ctx.cwd = projectName;
+    }
+
+    const isDirectoryEmpty =
```

Comment:
Should this be `isDirectoryNotEmpty` or `const isDirectoryEmpty = !(...);`? It seems to be truthy when the directory has files.

Reply:
haha yes good catch